src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -115,6 +115,15 @@ public Object instanceMain(final String[] args, final List<String> packageList,
      * @return the result of running  {program} with the given args, possibly null
      */
     protected static Object runCommandLineProgram(CommandLineProgram program, String[] rawArgs) {
+
+        /**
+         * The very first thing that any GATK application does is forces the JVM locale into US English, so that we don't have
+         * to think about number formatting issues.
+         *
+         * Placing the call here so that it is enabled for integration tests as well.
+         */
+        Utils.forceJVMLocaleToUSEnglish();
```

Comment:
It might be better to do this in a static block so the locale is established right from the start, and would be in place for unit tests that don't execute this code path.